cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -12,6 +12,26 @@ WORKER_NODE_ROOT="${CLUSTER_NAME}-worker"
 
 OPERATOR_GIT_HASH=8d3c30de8ec5a9a0c9eeb84ea0aa16ba2395cd68  # release-4.4
 
+function ensure_k8s_object {
+  object_type=$1
+  name=$2
+  namespace=$3
+
+  intervals=30
+  timeout=3
+
+  if [[ $namespace != "" ]];then
+    namespace="-n $namespace"
+  fi
```

Comment:
I think you should use `: ${namespace:=$3}`

Reply:
Can you please elaborate? (on which line, what is the difference)

Reply:
Instead of:
```
  local namespace=$3
  if [[ $namespace != "" ]];then
    namespace="-n $namespace"
  fi
```

You do:
```
: ${namespace:=$3}
```

Reply:
you mean?
```
  if [[ : ${namespace:=$3} != "" ]];then
    namespace="-n $namespace"
  fi
```

Reply:
No, I mean exactly what I wrote.

Reply:
Thanks for the explanations, 
it is necessary to append the `-n` flag.

I prefer to leave it as it is, because its easier to understand.